frontend/src/locales/en-US.json

```diff
@@ -431,7 +431,8 @@
       "completed": "completed",
       "failed": "failed",
       "task-name": " task {name}",
-      "committed-to-at": "committed {file} to{branch}{'@'}{repo}"
+      "committed-to-at": "committed {file} to{branch}{'@'}{repo}",
+      "dismissed-stale-approval": "dismissed stale approval of {task}"
```

Comment:
```suggestion
      "dismissed-stale-approval": "dismissed stale approvals of {task}"
```

Reply:
LGTM. Especially when we have a 2-step approval workflow in the future.